### ISSUE TITLE
Consolidate REST Request to One Service

### ISSUE THREAD

**Issue by dreamfarer** [sopra-fs25-group-20/sopra-fs25-group-20-client#101]
- just like gameAPI and stompAPI

**Comment by dreamfarer** [comment]
This refactor cannot be implemented due to time constraints. Previously unplanned usability and accessibility features take priority in order to provide a polished experience.
